frontend/src/actions/user.js

```diff
@@ -7,24 +7,24 @@ import { fetchGroup } from 'actions/group'
 
 // Registration
 export const requestUserRegistration = createAction('REQUEST USER REGISTRATION')
-export const receivedRegisteredUser = createAction('RECEIVED REGISTERED USER')
-export const userRegister = (username, password) => dispatch => {
+export const receiveRegisteredUser = createAction('RECEIVED REGISTERED USER')
+export const userRegister = (name, password) => dispatch => {
   dispatch(requestUserRegistration())
-  console.log(JSON.stringify({ name: username, password }))
   fetch(`${backend.protocol}://${backend.domain}:${backend.port}/users/register`, {
     ...defaultFetchParams,
     credentials:'include',
     method: 'POST',
-    body: JSON.stringify({ name: username, password })
+    body: JSON.stringify({ name, password })
   }).then(parseJSON)
     .then(res => {
-      if(res.success){
-        dispatch(receivedRegisteredUser())
-      }
-    })
+      dispatch(receiveRegisteredUser())
+      dispatch(userLogin(name, password))
+  }).catch(err => {
+    throw new Error(err)
+  })
 }
 
-//Logination
+// Logging in
```

Comment:
As for me not a good idea - we are returning to the concept sign in/sign out - logging in/logging out?
Please replace to more like login without in/out